CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 1.11.3 (Under active development)
 
+### AppCenter
+
+* **[Fix]** Handle some SQLite errors gracefully.
```

Comment:
Unless it's too complicated to explain, I'd suggest provide a brief description of what you are fixing in the changelog for the benefits of future references.

Reply:
The SDK was supposed to disable storing and sending logs when SQLite was failing instead of crashing the application. New SQLite APIs were used since version 1.9.0 and the new API exceptions were not caught, this is now fixed.

Or something like that.